src/Components/Email/Button/Form.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import React, { Fragment } from "react";
+import { FastField } from "formik";
+import Input from "elevate-ui/Input";
+import NumberIncrement from "elevate-ui/NumberIncrement";
+import ColorPicker from "elevate-ui/ColorPicker";
+import Select from "elevate-ui/Select";
+import Padding from "../../Common/Padding";
+
+import type { $SidebarProps } from "types";
+
+const fontFamilies = [
+  { label: "Arial", value: "Arial, Helvetica, sans-serif" },
+  { label: "Arial Black", value: "‘Arial Black’, Gadget, sans-serif" },
+  { label: "Bookman Old Style", value: "‘Bookman Old Style’, serif" },
+  { label: "Comic Sans MS", value: "‘Comic Sans MS’, cursive" },
+  { label: "Courier", value: "Courier, monospace" },
+  { label: "Courier New", value: "‘Courier New’, Courier, monospace" },
+  { label: "Garamond", value: "Garamond, serif" },
+  { label: "Georgia", value: "Georgia, serif" },
+  { label: "Impact", value: "Impact, Charcoal, sans-serif" },
+  { label: "Lucida Console", value: "‘Lucida Console’, Monaco, monospace" },
+  {
+    label: "Lucida Sans Unicode",
+    value: "‘Lucida Sans Unicode’, ‘Lucida Grande’, sans-serif",
+  },
+  { label: "MS Sans Serif", value: "‘MS Sans Serif’, Geneva, sans-serif;" },
+  {
+    label: "font-family: ‘MS Serif’",
```

Comment:
`font-family: ` 🤔

Reply:
haha good catch! i copied this from an example site and did not see that in it :)